Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py

```diff
@@ -155,17 +182,23 @@ def start_or_stop_subscription_request(self, content_type, start_or_stop_suffix)
             'Authorization': auth_string
         }
         params = {
+            'PublisherIdentifier': PUBLISHER_IDENTIFIER,
             'contentType': content_type
         }
-        response = self._http_request(
-            method='POST',
-            url_suffix=self.suffix_template.format(self.tenant_id, start_or_stop_suffix),
-            headers=headers,
-            params=params,
-            ok_codes=(200, 201, 202, 203, 204),
-            return_empty_response=True
-        )
-        return response
+        try:
```

Comment:
why do you need this code in addition to the modification of the wrapper function?
why can't you leave this function as is and catch the error only in the wrapper function?
is this function being used somewhere else?